ros2_batch_job/__main__.py

```diff
@@ -360,20 +360,30 @@ def build_and_test(args, job):
     # After Foxy, we introduced per-package changes to make local builds and CI
     # builds act the same.
     if args.ros_distro in ('dashing', 'eloquent', 'foxy'):
+        # xunit2 format is needed to make Jenkins xunit plugin 2.x happy
         with open('pytest.ini', 'w') as ini_file:
             ini_file.write('[pytest]\njunit_family=xunit2')
-        # check if packages have a pytest.ini file and add the xunit2
-        # format if it is not present
+        # check if packages have a pytest.ini file that doesn't choose junit_family=xunit2
+        # and patch configuration if needed to force the xunit2 value
+        pytest_6_or_greater = job.run([
```

Comment:
Looks like if this check fails, it fails the build ([for example](https://ci.ros2.org/job/ci_linux/11784/)).

Reply:
See https://github.com/ros2/ci/pull/498 for a proposed fix.